match/spec/encrypt_spec.rb

```diff
@@ -7,7 +7,7 @@
       File.write(@full_path, @content)
       @git_url = "https://github.com/fastlane/fastlane/tree/master/so_random"
       allow(Dir).to receive(:mktmpdir).and_return(@directory)
-      ENV["MATCH_PASSWORD"] = "my_pass"
+      ENV["MATCH_PASSWORD"] = '2"QAHg@v(Qp{=*n^'
```

Comment:
Does this change actually test something? Would the tests fail if the other changes in the PR were not here?